[2023-01-06 01:37:13,154] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: stock_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-06T01:21:29.019918+00:00 [queued]>
[2023-01-06 01:37:13,165] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: stock_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-06T01:21:29.019918+00:00 [queued]>
[2023-01-06 01:37:13,165] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 01:37:13,165] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-01-06 01:37:13,165] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-01-06 01:37:13,179] {taskinstance.py:1377} INFO - Executing <Task(DataprocCreateClusterOperator): create_cluster_dataproc_task> on 2023-01-06 01:21:29.019918+00:00
[2023-01-06 01:37:13,188] {standard_task_runner.py:52} INFO - Started process 9047 to run task
[2023-01-06 01:37:13,195] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'stock_data_pipeline_dag', 'create_cluster_dataproc_task', 'manual__2023-01-06T01:21:29.019918+00:00', '--job-id', '77', '--raw', '--subdir', 'DAGS_FOLDER/stock_data_pipeline_dag.py', '--cfg-path', '/tmp/tmpc0ajgkho', '--error-file', '/tmp/tmpo5mav6pq']
[2023-01-06 01:37:13,197] {standard_task_runner.py:80} INFO - Job 77: Subtask create_cluster_dataproc_task
[2023-01-06 01:37:13,266] {task_command.py:370} INFO - Running <TaskInstance: stock_data_pipeline_dag.create_cluster_dataproc_task manual__2023-01-06T01:21:29.019918+00:00 [running]> on host 7aa3417b2d87
[2023-01-06 01:37:13,402] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=stock_data_pipeline_dag
AIRFLOW_CTX_TASK_ID=create_cluster_dataproc_task
AIRFLOW_CTX_EXECUTION_DATE=2023-01-06T01:21:29.019918+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-06T01:21:29.019918+00:00
[2023-01-06 01:37:13,404] {dataproc.py:590} INFO - Creating cluster: move-stock-data
[2023-01-06 01:37:13,405] {base.py:68} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-01-06 01:39:54,215] {dataproc.py:534} INFO - Cluster created.
[2023-01-06 01:39:54,252] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=stock_data_pipeline_dag, task_id=create_cluster_dataproc_task, execution_date=20230106T012129, start_date=20230106T013713, end_date=20230106T013954
[2023-01-06 01:39:54,291] {local_task_job.py:156} INFO - Task exited with return code 0
